tests/foreman/ui/test_discoveryrule.py

```diff
@@ -160,7 +160,7 @@ def test_disable_discovery_rule_1(self):
             self.assertIsNotNone(self.discoveryrules.search(name))
 
     @run_only_on('sat')
-    def test_delete_discovery_rule_1(self):
+    def test_positive_delete_discovery_rule(self):
```

Comment:
I thought we are not repeating the entity name here.  Shouldn't this be just `test_positive_delete`?